src/components/learner-credit-management/data/hooks/useEnterpriseGroupMembersTableData.js

```diff
@@ -0,0 +1,78 @@
+import {
+  useCallback,
+  useMemo,
+  useState,
+} from 'react';
+import { camelCaseObject } from '@edx/frontend-platform/utils';
+import { logError } from '@edx/frontend-platform/logging';
+import debounce from 'lodash.debounce';
+
+import LmsApiService from '../../../../data/services/LmsApiService';
+import { transformGroupMembersTableResults } from '../utils';
+
+const useEnterpriseGroupMembersTableData = ({ groupId }) => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [enterpriseGroupMembersTableData, setEnterpriseGroupMembersTableData] = useState({
+    itemCount: 0,
+    pageCount: 0,
+    results: [],
+  });
+  const fetchEnterpriseGroupMembersData = useCallback((args) => {
+    const fetch = async () => {
+      try {
+        setIsLoading(true);
+        const options = {};
+        if (args?.filters.length > 0) {
+          options.user_query = args.filters[0].value;
+        }
+        if (args?.sortBy.length > 0) {
+          const sortByValue = args.sortBy[0].id;
+          if (sortByValue === 'memberDetails') {
+            options.sort_by = 'member_details';
```

Comment:
nit: can use [lodash snakeCase](https://lodash.com/docs/4.17.15#snakeCase) or @edx/frontend-platform/utils snakeCaseObject to convert.